jdbc/src/test/java/com/interface21/jdbc/core/JdbcTemplateTest.java

```diff
@@ -95,6 +204,75 @@ void updateForUpdateQuery() throws SQLException {
         assertThat(account).isEqualTo("updateGugu");
     }
 
+    @DisplayName("queryForObject(String, PreparedStatementSetter, RowMapper<>를 통해 select가능하다.")
+    @Test
+    void queryForObjectWithPreparedStatementSetter() {
+        // given
+        executeUpdateQuery("insert into users (account, password, email) values('gugu', '123', 'gugu@naver.com')");
+        executeUpdateQuery("insert into users (account, password, email) values('gugu2', '123', 'gugu2@naver.com')");
+        String selectQuery = "select * from users where account = ?";
+
+        // when
+        User user = jdbcTemplate.queryForObject(
+                selectQuery,
+                (rs) -> {
+                    rs.setString(1, "gugu");
+                },
+                userMapper);
+
+        // then
+        assertThat(user.email).isEqualTo("gugu@naver.com");
+    }
+
+    @DisplayName("queryForObject(String, PreparedStatementSetter, RowMapper<>)의 조회수가 1이아니면 예외가 발생한다")
```

Comment:
메서드 시그니처를 모두 사용하기 보다 메서드명만 사용해도 좋을 것 같아요! 메서드 시그니처 상 변동이 있을수도 있을 것 같아서요! 반영은 도도의 자유에 맡기겠습니다 :)

```suggestion
    @DisplayName("queryForObject의 조회수가 1이아니면 예외가 발생한다")
```